recipes/server.rb

```diff
@@ -61,12 +61,10 @@
 
 # Needed for the acme-client gem to continue connecting to pebble;
 # please do NOT do this on production Chef nodes!
+chef_cinc = node['chef_packages']['chef']['version'].to_i >= 15 ? 'cinc' : 'chef'
```

Comment:
I think this is a better name.
```suggestion
chef_path = node['chef_packages']['chef']['version'].to_i >= 15 ? 'cinc' : 'chef'
```